models/user.go

```diff
@@ -1355,6 +1356,138 @@ func GetWatchedRepos(userID int64, private bool) ([]*Repository, error) {
 	return repos, nil
 }
 
+// existsInSlice returns true if string exists in slice
+func existsInSlice(target string, slice []string) bool {
```

Comment:
Can you please move `existsInSlice` and `IsEqualSlice` to `modules/util/compare.go`?